reactnative/README.md

```diff
@@ -0,0 +1,66 @@
+# React Native Input Mask
```

Comment:
File name still shows as `reactnative`. I remember there being something tricky about changing just the letter casing of file names in git.